Sources/SnapshotAssertion/SnapshotAssertion.swift

```diff
@@ -72,9 +72,10 @@ extension String: Snapshot {
 public func assertSnapshot<S: Snapshot>(
   matching snapshot: S,
   identifier: String? = nil,
-  _ file: StaticString = #file,
-  _ function: String = #function,
-  _ line: UInt = #line)
+  pathExtension: String? = nil,
```

Comment:
nice nice! so this will allow me to override extension when doing a plain text snapshot?